flink/v1.12/flink/src/main/java/com/netease/arctic/flink/write/ArcticFileWriter.java

```diff
@@ -148,16 +148,7 @@ private void initTaskWriterFactory(Long mask) {
   }
 
   private Long getTransactionId() {
-    Long transaction;
-    if (table.isKeyedTable()) {
-      String signature = BaseEncoding.base16().encode((jobId + checkpointId).getBytes());
-      transaction = table.asKeyedTable().beginTransaction(signature);
-      LOG.info("table:{}, signature:{}, transactionId:{}. From jobId:{}, ckpId:{}", table.name(), signature,
-          transaction, jobId, checkpointId);
-    } else {
-      transaction = null;
-    }
-    return transaction;
+    return null;
```

Comment:
The variable transactionId is always null in the flink writer operator lifecycle, so this method and [`setTransactionId`](https://github.com/NetEase/arctic/blob/master/flink/v1.12/flink/src/main/java/com/netease/arctic/flink/write/ArcticRowDataTaskWriterFactory.java#L56) should be removed because they seemed redundant.

Reply:
OK, I will remove it